Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/AuditLogging-Passthrough-soapui-project.xml

```diff
@@ -21355,6 +21411,54 @@ nhinc.FileUtils.updateProperty(context.findProperty('GatewayPropDir'),context.fi
       <con:name>ZipCode</con:name>
       <con:value>12345</con:value>
     </con:property>
+    <con:property>
+      <con:name>MessageID</con:name>
```

Comment:
I took one of the sample messageID value that we used in our standard Connect Validation suite. As per my understanding, In the code we take the value that we passed in and format before sending request. But I can change if required.